cdl/dtd_service.go

```diff
@@ -0,0 +1,107 @@
+package cdl
+
+import (
+	"fmt"
+	"github.com/go-playground/validator/v10"
+	"io"
+	"net/http"
+	"path"
+)
+
+type DataTypeDefinition struct {
+	ID string `json:"id,omitempty"`
+	Name string `json:"name,omitempty"`
+	Description string `json:"description,omitempty"`
+	JsonSchema interface{} `json:"jsonSchema,omitempty"`
```

Comment:
instead of `interface{}` you can use the `json.RawMessage` type here

Reply:
Now using json.RawMessage for jsonSchema field of DTD